### PR TITLE
SDK-365: Release 2.3.2

### PR DESCRIPTION
**Pull request by Aditi3**
- Added support for Push Notification Click Callbacks.
- Added CleverTap iOS SDK dependency via CocoaPods in the plugin.xml
- Update for CleverTap Android SDK v4.0.2
- Updated documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+Version 2.3.2 *(06 January, 2021)*
+-------------------------------------------
+* Added support for Push Notification Click Callbacks.
+* Added CleverTap iOS SDK dependency via CocoaPods in the plugin.xml
+* Update for CleverTap Android SDK v4.0.1
```
Comment (darshanclevertap) [inline CHANGELOG.md:8]: @kumaratulismu It should show Android SDK v4.0.2

File: ExampleProject/platforms/android/app/build.gradle [hunk 2]
```diff
@@ -351,9 +351,9 @@ dependencies {
 
     // SUB-PROJECT DEPENDENCIES START
     implementation(project(path: ":CordovaLib"))
-    implementation "com.google.firebase:firebase-messaging:20.2.4"
     implementation "com.android.support:support-v4:28.0.0"
-    implementation "com.clevertap.android:clevertap-android-sdk:4.0.0"
+    implementation "com.google.firebase:firebase-messaging:20.2.4"
+    implementation "com.clevertap.android:clevertap-android-sdk:4.0.1"
```
Comment (darshanclevertap) [inline ExampleProject/platforms/android/app/build.gradle:356]: @kumaratulismu Update to v4.0.2

**Review by darshanclevertap — APPROVED** [review]
Reviewed changes by @kumaratulismu 
LGTM 👍🏾

**Review by kumaratulismu — APPROVED** [review]
Looks good.

**Pull request merged**